dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyConnectionClient.java

```diff
@@ -90,6 +95,8 @@ protected void initConnectionClient() {
         this.closePromise = new DefaultPromise<>(GlobalEventExecutor.INSTANCE);
         this.init = new AtomicBoolean(false);
         this.increase();
+        this.isReconnecting = new AtomicBoolean(false);
+        scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
```

Comment:
This executor should be managed by `org.apache.dubbo.common.threadpool.manager.FrameworkExecutorRepository`